<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggedInWindow</class>
 <widget class="QDialog" name="LoggedInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brainstorm | Logged in</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="pageUser">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Welcome back!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageAdmin">
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
font-weight: bold;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Admin permissions: Enabled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>520</y>
       <width>82</width>
       <height>116</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAddGrade">
        <property name="text">
         <string>Add Grade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonUpdateGrade">
        <property name="text">
         <string>Update Grade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDeleteGrade">
        <property name="text">
         <string>Delete Grade</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>370</y>
       <width>521</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="rowCount">
      <number>8</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Monday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Thursday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Friday</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButtonAddPerms">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>340</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add perms</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddToGrade">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>340</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to class</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Welcome back!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageChemistry">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>521</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="rowCount">
      <number>8</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Monday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Thursday</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Friday</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>9A</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>9V</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>10G</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>10A</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>10A</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>9A</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>9V</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>10G</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>10A</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>9G</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>9B</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>10G</string>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string>9G</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>9B</string>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>10V</string>
      </property>
     </item>
     <item row="3" column="3">
      <property name="text">
       <string>10V</string>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>10A</string>
      </property>
     </item>
     <item row="4" column="2">
      <property name="text">
       <string>10V</string>
      </property>
     </item>
     <item row="4" column="4">
      <property name="text">
       <string>10B</string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>10G</string>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string>10V</string>
      </property>
     </item>
     <item row="5" column="4">
      <property name="text">
       <string>10B</string>
      </property>
     </item>
     <item row="6" column="3">
      <property name="text">
       <string>10B</string>
      </property>
     </item>
     <item row="7" column="3">
      <property name="text">
       <string>10B</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>82</width>
       <height>116</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonAddGrade_2">
        <property name="text">
         <string>Add Grade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonUpdateGrade_2">
        <property name="text">
         <string>Update Grade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDeleteGrade_2">
        <property name="text">
         <string>Delete Grade</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QStackedWidget" name="stackedWidgetChemistry">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>290</y>
       <width>531</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Welcome back!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>160</y>
       <width>82</width>
       <height>116</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonAddAbsence">
        <property name="text">
         <string>Add Absence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonUpdateAbsence">
        <property name="text">
         <string>Update Absence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDeleteAbsence">
        <property name="text">
         <string>Delete Absence</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
